lib/gwdb-versioned/gwdb_driver.ml

```diff
@@ -130,8 +130,7 @@ end = struct
     if not (directory_exists base) then create_files base;
     let patchfile = D.patch_file base in
     let patchfile_tmp = D.tmp_file patchfile in
-    if Sys.file_exists patchfile_tmp then
-      failwith "Error while writing patch file : temporary file remained";
+    if Files.exists patchfile_tmp then Files.rm patchfile_tmp;
```

Comment:
```suggestion
    Files.rm patchfile_tmp;
```

because:

https://github.com/geneweb/geneweb/blob/0b8d3d5c0c16a01dc7de1acfa3d8fb9da5f74337/lib/system_util/files.ml#L2-L5